src/manifests_workflow/component.py

```diff
@@ -15,10 +18,18 @@ def __init__(self, name, repo, snapshot=False, checks=[]):
         self.checks = checks
 
     @classmethod
-    def gradle_cmd(self, target, props={}):
-        cmd = [f"./gradlew {target}"]
-        cmd.extend([f"-D{k}={v}" for k, v in props.items()])
-        return " ".join(cmd)
+    def branches(self, uri):
+        branches = ["main"]
+        remote_branches = (
+            subprocess.check_output(
+                f"git ls-remote {uri} refs/heads/* | cut -f2 | cut -d/ -f3", shell=True
+            )
+            .decode()
+            .split("\n")
+        )
+        # return "main" and "x.y" branches only
```

Comment:
This comment is misleading because main is included at line 22, I'd recommend splitting the comments or add a function comment that indicates what should be expected inside of branches instead of the inline comment